src/utils/settings.ts

```diff
@@ -117,14 +117,16 @@ export const migrateDeprecatedSettings = (): Settings => {
 
   if (
     settings.network.explorerApiHost === 'https://mainnet-backend.alephium.org' ||
-    settings.network.explorerApiHost === 'https://backend-v18.mainnet.alephium.org'
+    settings.network.explorerApiHost === 'https://backend-v18.mainnet.alephium.org' ||
```

Comment:
@mvaivre, please, don't mind the network settings migration changes. We worked together on updating how the network settings migrate in the v2.0 branch. For now, I simply updated the existing migration code, since it will soon be replaced with the [new one](https://github.com/alephium/desktop-wallet/blob/v2.0/src/utils/migration.ts#L141-L158) with v2.0.

Reply:
Makes sense!